app/models/spree/payment/processing.rb

```diff
@@ -292,6 +294,11 @@ def check_environment
       def gateway_order_id
         "#{order.number}-#{identifier}"
       end
+
+      # TODO: Remove check once :captured_at migration complete
+      def captured_at_migration_complete?
+        respond_to?(:captured_at=)
+      end
```

Comment:
I think we can just drop this construct altogether. If this PR gets merged and then included in a release, the migration will be run during deployment and the new code will be applied slightly after it's completed. So there shouldn't be a point where the new code is trying to write to a non-existent database column :ok_hand:

Reply:
Okay good to know, I have removed that now.